spec/tapioca/cli/dsl_spec.rb

```diff
@@ -1590,6 +1590,102 @@ def bar(&block); end
           refute_success_status(result)
         end
       end
+
+      describe "environment" do
+        before(:all) do
+          @project.tapioca("init")
+
+          @project.write("lib/post.rb", <<~RB)
+            require "smart_properties"
```

Comment:
I wasn't sure why we needed `smart_properties` in this test? I guess nothing is happening is we don't have a real DSL to trigger RBI generation?

Reply:
Yes, that's the reason, I wanted to make sure that the RBIs generated were different if different environments defined constants differently. Alternatively I could have used a custom DSL compiler, but Smart Properties is just easier to use.